File: Java/2185_Counting_Words_With_a_Given_Prefix.java
class Solution {
    public int prefixCount(String[] words, String pref) {
        int result = 0;
        for(String word : words){
            if(word.startsWith(pref)){
                result++;
            }
            /*
            if(word.length() >= pref.length()){
                if(word.substring(0, pref.length()).equals(pref)){
                    result++;
                }
            }
            */
        }
        return result;
    }
}

File: CS/2185_Counting_Words_With_a_Given_Prefix.cs
public class Solution {
    public int PrefixCount(string[] words, string pref) {
        return words.Where(x => x.StartsWith(pref)).Count();
        /*
        int result = 0;
        foreach(string word in words){
            result += word.StartsWith(pref) ? 1 : 0;
        }
        return result;
        */
    }
}
